system/lib/compiler-rt/sanitizer-common.symbols

```diff
@@ -1,1392 +0,0 @@
-
```

Comment:
Can you delete these .symbols files now?

Reply:
They have been deleted.